src/languages.ts

```diff
@@ -130,24 +130,29 @@ export const languages: {
         close: '"'
       },
       {
-        open: "'",
-        close: "'"
```

Comment:
From what I gathered from your comment, this is a string (or character) in Julia, right?
This was added to avoid matching any word inside strings and such. Maybe the single quote should be kept?

Reply:
Thank you for your reply. In [Julia doc](https://docs.julialang.org/en/v1/manual/strings/#man-characters-1), `'`s are only reserved for characters. A string inside `''` is illegal.
```julia
julia> 'end'
ERROR: syntax: invalid character literal
Stacktrace:
 [1] top-level scope at none:0
```

Reply:
I see! So there should be no problem removing this